### DISCUSSION TITLE
中文特化版为什么不支持英文缩写

### DISCUSSION THREAD

**Discussion by winwindg** [fishaudio/Bert-VITS2#352] [General]
比如AI、LED、SUV这种常见单词，用了中文特化版都被吞掉了，现在主播中英文混合不是很常见吗？

如果要基于中文特化版改成支持英文，看了一下代码，挺麻烦的，1.3B生成的是2048的张量，英文的是1024的张量，两个不匹配。

**Comment by winwindg** [comment]
如果用2.3，很多新功能又没了。。。

**Comment by Stardust-minus** [comment]
你猜猜为什么叫中文特化

**Comment by winwindg** [comment]
如果这样，就只能玩玩了，不能真正用于交付

**Comment by v3ucn** [comment]
可以改成使用2.3的模型推理英文，用中文特化的模型推理中文，修改webui.py就可以了

**Reply by winwindg** [reply]
怎么改，要把中文特化版的哪些文件替换为2.3的文件？data_utils.py和models.py ...
